src/routes/sessions/[uuid]/.SDKHeader.svelte

```diff
@@ -0,0 +1,3 @@
+<svelte:head>
+  <script crossorigin src="https://unpkg.com/webex@^2/umd/webex.min.js"></script>
+</svelte:head>
```

Comment:
This component doesn't add any visual elements to the page - expect for importing another library over CDN. This may be inlined - unless the SDK is being imported at many places.

Reply:
svelte:head can not stay in a block. Therefore, we need to add move the logic in a component for conditional rendering.

Reply:
Sure but it's possible to use condition inside it, while inlining. Example:
```
<svelte:head>
    {#if isRelEnabled}
        <link rel="stylesheet" href="/stylesheets/{selected}.css" />
    {/if}
</svelte:head>
```

Reply:
create a JIRA to import webex node module